homework-submission/week1/src/Title.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react';
+class Title extends Component {
+  render() {
+    return (
+      <React.Fragment>
+        <h1>Todo List</h1>
+      </React.Fragment>
+    );
```

Comment:
I am happy that you try to use React.Fragment, could you explain why did you use it here? what the reason for it? 
I believe it is not needed, but it works